synapse/lib/layer.py

```diff
@@ -1101,27 +1117,27 @@
     def __repr__(self):
         return f'Layer ({self.__class__.__name__}): {self.iden}'
 
-    async def __anit__(self, layrinfo, dirn, nexsroot=None, allow_upstream=True, mapasync=True, maxreplaylog=10000):
+    async def __anit__(self, core, layrinfo):
```

Comment:
An edge case probably, but maybe we want to prevent cloning a mirrored layer so we don't have two mirror loops running?